### PR TITLE
Doc : FR : Update of chapter 4 xc tasks

### PR DESCRIPTION
**Pull request by mlep**
Major update of the French translation of chapter 4 (Cross-country tasks), based on new English version (PR #309).

### CODE REVIEW

File: doc/manual/fr/ch04_xc_tasks.tex [hunk 1]
```diff
@@ -1,543 +1,682 @@
+% !TeX encoding = utf8
+% !TeX spellcheck = fr
+
 \chapter{Circuits}\label{cha:tasks}
 
-XCSoar fournit toutes les fonctionnalités de gestion des circuits sur la campagne. Ils peuvent être édités avant le décollage et modifiés en vol. Le passage au point de virage suivant est automatique. Ce chapitre décrit aussi l'utilisation de l'enregistrement et relecture des vols au format IGC.
+\xc{} fournit un système complet de fonctionnalités de gestion des circuits sur la campagne. Ils peuvent être
+édités avant le décollage et, lors d'une vol d'agrément
```
Comment (lordfolken) [inline doc/manual/fr/ch04_xc_tasks.tex]: d'un vol d'agrément

File: doc/manual/fr/ch04_xc_tasks.tex [hunk 2]
```diff
@@ -1,543 +1,682 @@
...
+sur la campagne, être modifiés en vol. Le passage au point de virage suivant est automatique,
+mais peut aussi être effectué manuellement. Beaucoup des calculs d'\xc{} sont en lien avec
```
Comment (lordfolken) [inline doc/manual/fr/ch04_xc_tasks.tex]: de calculs

File: doc/manual/fr/ch04_xc_tasks.tex [hunk 3]
```diff
@@ -1,543 +1,682 @@
...
-\item[Diff Alt MC sécurité] Différence d'altitude nécessaire comparée au calage MC de sécurité. (Voir \ref{sec:secu-parameter} )
-\item[Diff. Alt. MC act] Différence d'altitude nécessaire comparée au calage MC actuel.
+\item[Diff. Alt. MC 0] Différence d'altitude avec un calage MacCready à zéro.
+\item[Diff Alt MC sécurité] Différence d'altitude avec le calage MacCready de 
+sécurité (voir~\ref{sec:secu-parameter}).
+\item[Diff. Alt. MC act.] Différence d'altitude au calage MacCready actuel.
 \end{description}
 
-La page des détails des points de virage est constituées de deux pages (accessibles avec les boutons \button{$>$} et \button{$<$} ). En fonction de l'existence et du nombre d'autres détails propre au point de virage visualisé, d'autres pages pourront être affichées.
+La fenêtre des détails des points de virage est constituées de deux pages, accessibles avec les touches
```
Comment (lordfolken) [inline doc/manual/fr/ch04_xc_tasks.tex]: constituée

Comment (mlep) [reply doc/manual/fr/ch04_xc_tasks.tex]: Corrected.

File: doc/manual/fr/ch04_xc_tasks.tex (hunk 2 above)
Comment (mlep) [reply doc/manual/fr/ch04_xc_tasks.tex]: In fact, in French, both are valid.

File: doc/manual/fr/ch04_xc_tasks.tex (hunk 1 above)
Comment (mlep) [reply doc/manual/fr/ch04_xc_tasks.tex]: Corrected

**Comment by mlep** [comment]
@lordfolken: Thank you for reviewing my changes and pointing out some errors!
They are now corrected

**Pull request merged**